examples/async_fl/async_aggregator.py

```diff
@@ -290,11 +300,17 @@ def client_completion_handler(self, results):
         # Format:
         #       -results = {'clientId':clientId, 'update_weight': model_param, 'moving_loss': round_train_loss,
         #       'trained_size': count, 'wall_duration': time_cost, 'success': is_success 'utility': utility}
+
+        # [Async] some clients are scheduled earlier, which should be aggregated in previous round but receive the result late
```

Comment:
For my understanding: why do we want to ignore clients that should be aggregated in previous rounds? Don't we want to aggregate it anyways with a staleness factor?

Reply:
Yea, that's also a solution🤔, if we ignore the fact that the supposed `end_time` of the result has passed. 
(The reason of receive "past" results is that we schedule the training task based on its `end_time`, but we cannot control the order of task finishing. )
I will either fix the grpc problem or just follow your suggestion. @fanlai0990 what do you think

Reply:
I think we can aggregate it according to the staleness factor for now.